pmpsui/pmps.py

```diff
@@ -86,6 +93,8 @@ def __init__(self, parent=None, args=None, macros=None):
         # add CFG macro to display the Line when starting without macros
         macros['CFG'] = config_name
         super(PMPS, self).__init__(parent=parent, args=args, macros=macros)
+        # Move to same location as main screen
+        self.splash.move(self.pos())
```

Comment:
this didn't work for me, it moved the splash screen to the top-left corner of the screen it spawned on rather than over to the other screen that contains the main window

Reply:
I'm still pretty confused as to why the pydm window and the splash screen pick different screens to open on, how can this happen?

Reply:
Does the main window start there then move by chance? I don't understand why this would place the splash anywhere but on top of the main window

Reply:
From https://doc.qt.io/qt-6/qsplashscreen.html:

> The splash screen appears in the center of the screen

It's appearing slightly off to the right of my larger monitor, effectively in the "center" of the combined 2-screen setup

The main windows starts near the top-left of the "primary" monitor and stays there